modules/reissuing-internal-certificates-for-secured-clusters-by-using-automatic-certificate-renewal.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="reissuing-internal-certificates-for-secured-clusters-by-using-automatic-certificate-renewal_{context}"]
+= Reissuing internal certificates for secured clusters by using automatic certificate renewal
+
+The secured cluster components and Central can communicate between each other by using the certificate-based architecture (mTLS). This feature enables the automatic rotation of certificates. {rh-rhacs-first} components use these certificates on your secured clusters and require no user interaction.
```

Comment:
A few nitpicks here:
- we could drop the first sentence (because it repeats something we already state in the parent section)
- sorry for changing my mind on this one, but maybe it's not ideal that we say that it requires no user interaction, and then we proceed to give instructions on how to manually apply the certificates :)